### PR TITLE
update order api with test cases

### PR DESCRIPTION
**Pull request by ankitdas13**

### CODE REVIEW

File: src/test/java/com/razorpay/OrderClientTest.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+    private static final String ORDER_ID = "order_EKwxwAgItmmXdp";
+
+    /**
+     * Create order with basic details such as currency and amount details
+     * @throws RazorpayException
+     */
+    @Test
+    public void create() throws RazorpayException {
+        JSONObject request = new JSONObject("{amount:50000,currency:\"INR\",receipt:\"receipt#1\",notes:{key1:\"value3\",key2:\"value2\"}}");
```
Comment (ersinghlovepreet) [inline src/test/java/com/razorpay/OrderClientTest.java:25]: we can format it, for better readability

File: src/test/java/com/razorpay/OrderClientTest.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    }
+
+    /**
+     * Update an order of respective customer using order id with object of that properties
+     * @throws RazorpayException
+     */
+    @Test
+    public void edit() throws RazorpayException {
+        JSONObject request = new JSONObject("{\"notes\":{\"key1\":\"value3\",\"key2\":\"value2\"}}");
+        String mockedResponseJson = "{\"id\":"+ORDER_ID+",\"entity\":\"order\",\"amount\":2200,\"amount_paid\":0,\"amount_due\":2200,\"currency\":\"INR\",\"receipt\":\"Receipt#211\",\"offer_id\":null,\"status\":\"attempted\",\"attempts\":1,\"notes\":{\"notes_key_1\":\"Tea,EarlGrey,Hot\",\"notes_key_2\":\"Tea,EarlGrey…decaf.\"},\"created_at\":1572505143}";
```
Comment (ersinghlovepreet) [inline src/test/java/com/razorpay/OrderClientTest.java:109]: some jsons string are used in multiple methods, can create private method

**Review by ersinghlovepreet — CHANGES_REQUESTED** [review]

File: src/test/java/com/razorpay/OrderClientTest.java (hunk 1 above)
Comment (ankitdas13) [reply src/test/java/com/razorpay/OrderClientTest.java:25]: there are some jsons are very long are we format them also ?

File: src/test/java/com/razorpay/OrderClientTest.java (hunk 2 above)
Comment (ankitdas13) [reply src/test/java/com/razorpay/OrderClientTest.java:109]: ok i will do that .

File: src/test/java/com/razorpay/OrderClientTest.java (hunk 1 above)
Comment (ersinghlovepreet) [reply src/test/java/com/razorpay/OrderClientTest.java:25]: yeah for beter readability, it should be formated

**Review by ersinghlovepreet — CHANGES_REQUESTED** [review]

**Review by ersinghlovepreet — APPROVED** [review]

**Pull request merged**